stacker_blueprints/empire/controller.py

```diff
@@ -0,0 +1,177 @@
+from troposphere import (
+    Ref,
+    Output,
+    GetAtt,
+    FindInMap,
+)
+from troposphere import (
+    ec2,
+    autoscaling,
+    ecs,
+)
+from troposphere.autoscaling import Tag as ASTag
+from troposphere.iam import (
+    InstanceProfile,
+    Policy,
+    Role,
+)
+
+from awacs.helpers.trust import (
+    get_default_assumerole_policy,
+)
+
+from .base import EmpireBase
+
+from .policies import ecs_agent_policy
+
+CLUSTER_SG_NAME = "EmpireControllerSecurityGroup"
+
+
+class EmpireController(EmpireBase):
+    PARAMETERS = {
+        "VpcId": {
+            "type": "AWS::EC2::VPC::Id",
+            "description": "Vpc Id"},
+        "DefaultSG": {
+            "type": "AWS::EC2::SecurityGroup::Id",
+            "description": "Top level security group."},
+        "PrivateSubnets": {
+            "type": "List<AWS::EC2::Subnet::Id>",
+            "description": "Subnets to deploy private instances in."},
+        "AvailabilityZones": {
+            "type": "CommaDelimitedList",
+            "description": "Availability Zones to deploy instances in."},
+        "InstanceType": {
+            "type": "String",
+            "description": "Empire AWS Instance Type",
+            "default": "m3.medium"},
+        "MinHosts": {
+            "type": "Number",
+            "description": "Minimum # of empire minion instances.",
+            "default": "2"},
+        "MaxHosts": {
+            "type": "Number",
+            "description": "Maximum # of empire minion instances.",
+            "default": "3"},
+        "SshKeyName": {
+            "type": "AWS::EC2::KeyPair::KeyName"},
+        "ImageName": {
+            "type": "String",
+            "description": (
+                "The image name to use from the AMIMap (usually found in the"
+                " config file.)"
```

Comment:
i was being inconsistent before, now i'm trying to be more consistent. the logic for the space being on the next line is so that you can delete the line and not have to adjust the trailing space on the line above. i don't have a preference on this though, just want to be consistent across blueprints.